multiqc/utils/config.py

```diff
@@ -22,18 +22,18 @@
 
 # Get the MultiQC version
 version = pkg_resources.get_distribution("multiqc").version
-cwd = os.getcwd()
 script_path = os.path.dirname(os.path.realpath(__file__))
 git_hash = None
 git_hash_short = None
 try:
-    os.chdir(script_path)
-    git_hash = subprocess.check_output(['git', 'rev-parse', 'HEAD'], stderr=subprocess.STDOUT)
+    git_hash = subprocess.check_output( ['git', 'rev-parse', 'HEAD'],
+                                        cwd=script_path,
```

Comment:
Nice! This is much cleaner. I can't find this documented for `check_output` anywhere, only `Popen`. If it works then that's great though!